packages/rpc-graphql/src/__tests__/__setup__.ts

```diff
@@ -1980,6 +1980,66 @@ export const mockTransactionToken2022AllExtensions = {
                     programId: 'TokenzQdBNbLqP5VEhdkAS6EPFLC1PHnBqCXEpPxuEb',
                     stackHeight: null,
                 },
+                {
+                    parsed: {
+                        info: {
+                            account: '2Pwe6Yahh5cbzvCwRMtTYFeboSwYiWeHhYJzZZBsU6eB',
+                            owner: 'FsHcsGiY43QmZc6yTgwYC1DA5U3ZgycXxn3bd2oBjrEZ',
+                            multisigAuthority: null,
+                            signers: null,
```

Comment:
We don't want these `null` values in the mocked RPC response payloads, just the Jest checks in the unit tests themselves.